orion-server/src/main/resources/webapp/src/actions/cluster.js

```diff
@@ -30,6 +30,9 @@ export const UTILIZATION_REQUESTED = "UTILIZATION_REQUESTED";
 export const UTILIZATION_RECEIVED = "UTILIZATION_RECEIVED";
 export const COST_REQUESTED = "COST_REQUESTED";
 export const COST_RECEIVED = "COST_RECEIVED";
+export const AMILIST_REQUESTED = "AMILIST_REQUESTED";
+export const AMILIST_RECEIVED = "AMILIST_RECEIVED";
+export const AMITAG_UPDATE = "AMITAG_UPDATE";
```

Comment:
nit: use `AMI_LIST_` instead of `AMILIST`? same for `AMITAG`

Reply:
Done